src/components/InsureeSearcher.js

```diff
@@ -45,6 +45,7 @@ class InsureeSearcher extends Component {
     );
     this.defaultPageSize = props.modulesManager.getConf("fe-insuree", "insureeFilter.defaultPageSize", 10);
     this.locationLevels = this.props.modulesManager.getConf("fe-location", "location.Location.MaxLevels", 4);
+    this.isWorker = props.modulesManager.getConf("fe-insuree", "isWorker", false);
```

Comment:
Same here, consider moving the default value to the constants.

Reply:
Ideally, we should keep the rest of the default values as constants, but I'm aware that there are plenty of them. There is no need to refactor this in the scope of this ticket.

Reply:
done